plugins/node/opentelemetry-instrumentation-connect/package.json

```diff
@@ -41,16 +41,16 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.0.1"
+    "@opentelemetry/api": "^1.0.2"
```

Comment:
Do we want to update the peer dependency? v1.0.1 is still valid to be used with the instrumentation to my understanding.

Reply:
Yes it should be ok. I wish renovate was easier to configure. I'm not at all confident it will be easy to make that happen but i'll give it a shot